_ng/server/sub_generators_endpoint.js

```diff
@@ -38,4 +38,26 @@ exports.EndpointSubGenerator = class EndpointSubGenerator {
         case ServerFactory.tokens().NODE: NodeFactory.build(this.wrapper).copyFiles(); break;
     }
   }
-}
+
+  end() {
+    const routesFile = 'server/routes/index.js';
+    const newRoute = `${knownPaths.PATH_SERVER_FEATURES + this.wrapper.feature}/route/${this.wrapper.name}-route`;
+    const reqPath = this.wrapper.relativeRequire(newRoute, routesFile);
+    const routeImportConfig = {
+      file: routesFile,
+      needle: '// Import routes below',
+      splicable: [
+        `import ${this.wrapper.name}Routes from '${reqPath}';`
+      ]
+    };
+    const routeInitConfig = {
+      file: routesFile,
+      needle: '// Init routes below',
+      splicable: [
+        `${this.wrapper.name}Routes.init(router);`
```

Comment:
This is also different for Go, and not only that, the types are different (echo vs gin).